flang/lib/Lower/Bridge.cpp

```diff
@@ -290,27 +290,28 @@ class FirConverter : public Fortran::lower::AbstractConverter {
   fir::ExtendedValue genExprAddr(const Fortran::lower::SomeExpr &expr,
                                  Fortran::lower::StatementContext &context,
                                  mlir::Location *loc = nullptr) override final {
-    return createSomeExtendedAddress(loc ? *loc : toLocation(), *this, expr,
-                                     localSymbols, context);
+    return Fortran::lower::createSomeExtendedAddress(
+        loc ? *loc : toLocation(), *this, expr, localSymbols, context);
   }
   fir::ExtendedValue
   genExprValue(const Fortran::lower::SomeExpr &expr,
                Fortran::lower::StatementContext &context,
                mlir::Location *loc = nullptr) override final {
-    return createSomeExtendedExpression(loc ? *loc : toLocation(), *this, expr,
-                                        localSymbols, context);
+    return Fortran::lower::createSomeExtendedExpression(
```

Comment:
A search for the string `createSome` in header and code files gets 39 hits on the 6 distinct names on the left in this list:

-   `createSomeArrayAssignment` -> `createArrayAssignment`
-   `createSomeArrayBox` -> `createArrayBox`
-   `createSomeArrayTempValue` -> `createArrayTempValue`
-   `createSomeExtendedAddress` -> `createExtendedAddress`
-   `createSomeExtendedExpression` -> `createExtendedExpression`
-   `createSomeInitializerExpression` -> `createInitializerExpression`

There are no hits on the corresponding strings on the right in the list.

Most of these functions have a `SomeExpr` argument, which may have influenced the name choice.  But there a number of other functions with `SomeExpr` arguments that do not have `Some` in their names.  I wonder if the word `Some` in these names may not be useful?

Reply:
The "Some" part of the name is vestigial. I've been slowly phasing it out.